masonry-go.go

```diff
@@ -158,6 +159,35 @@ func NewCLIApp() *cli.App {
 				},
 			},
 		},
+		{
+			Name:    "diff",
+			Aliases: []string{"d"},
+			Usage:   "Compute Gap Analysis",
+			Flags: []cli.Flag{
+				cli.StringFlag{
+					Name:        "opencontrols, o",
+					Value:       "opencontrols",
+					Usage:       "Set opencontrols directory",
+					Destination: &opencontrolDir,
+				},
+			},
+			Action: func(c *cli.Context) {
+				config := inventory.Config{
+					Certification:  c.Args().First(),
+					OpencontrolDir: opencontrolDir,
+				}
+				inventory, err := inventory.ComputeGapAnalysis(config)
+				if err != nil && len(err) > 0 {
+					fmt.Println(strings.Join(err, "\n"))
+					os.Exit(1)
+				}
+
+				fmt.Printf("\nNumber of missing controls: %d\n", len(inventory.MissingControlList))
+				for standardAndControl, _ := range inventory.MissingControlList {
+					fmt.Println(standardAndControl)
+				}
```

Comment:
Since it doesn't seem easy to test this file, maybe move all of this into a standalone function where we can test the output?